.github/workflows/e2e.yml

```diff
@@ -53,6 +53,8 @@ jobs:
         if: matrix.extension == 'sns'
       - name: run test
         run: timeout 2400 e2e/bats/bin/bats extensions/${{ matrix.extension }}/e2e/tests/*.bash
+      - name: run sns tests
+        run: timeout 2400 e2e/launch-simple-sns.sh
```

Comment:
NCR but GitHub actions explicitly supports timeouts:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepstimeout-minutes

Reply:
Thanks for the pointer. I was following the pattern I found in the CI config file. I'll also consult with the SDK team to find out their view on how to add this script to the CI.